src/Uno.Toolkit.UI/Helpers/ScrollableHelper.cs

```
@@ -57,7 +57,7 @@ private static async Task ScrollTo(this ListViewBase lvb, object item, Func<Depe
 			bool isVirtualizing = default(bool);
 			double previousHorizontalOffset = default(double), previousVerticalOffset = default(double);
 
-			var scrollViewer = lvb.GetFirstDescendant<ScrollViewer>();
+			var scrollViewer = lvb.GetFirstDescendant<ScrollViewer>()!;
```

should be a `?? throw` InvalidOperationEx here